code/modules/reagents/reagent_containers/dropper.dm

```diff
@@ -9,13 +9,21 @@
 	amount_per_transfer_from_this = 5
 	possible_transfer_amounts = list(1,2,3,4,5)
 	volume = 5
-	var/filled = 0
+
+/obj/item/weapon/reagent_containers/dropper/on_reagent_change()
+	update_icon()
+
+/obj/item/weapon/reagent_containers/dropper/update_icon()
+	if(reagents.total_volume<=0)
+		icon_state = "[initial(icon_state)]"
+	else
+		icon_state = "[initial(icon_state)]1"

```

Comment:
I did it out of consistency only, for example: https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/reagents/reagent_containers/bottle.dm#L15:L44 and https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/reagents/reagent_containers/syringes.dm#L28

While yeah it doesn't make much sense to have a proc just call another proc, other similar things do it to so I figured there would be a reason for it, and also for consistency